<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>enseignant</class>
 <widget class="QDialog" name="enseignant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>501</width>
     <height>671</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>720</y>
     <width>139</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Pictures/empty_recycle_bin_by_brightknight.png</normaloff>../../Pictures/empty_recycle_bin_by_brightknight.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>720</y>
     <width>140</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Afficher</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>421</width>
     <height>751</height>
    </rect>
   </property>
   <property name="title">
    <string>Enseignant</string>
   </property>
   <widget class="QPushButton" name="pb_ajouter">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>340</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Pictures/Male-user-add-icon.png</normaloff>../../Pictures/Male-user-add-icon.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_salaire">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>SALAIRE :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Salaire">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>280</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_CIN">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_CIN">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CIN :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Matiere">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>250</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Matiere">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>MATIERE :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_prenom">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Prenom">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>PRENOM :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nom">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_NOM">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>NOM :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_modifier">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>340</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Modifier</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Pictures/flèche-de-cycle-renouvellement-écolo-1560x1502.png</normaloff>../../Pictures/flèche-de-cycle-renouvellement-écolo-1560x1502.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>690</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Mail">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mail : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_MDP">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mot de Passe : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Mail">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_MDP">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_date">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Date de naissance :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Date">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_nationalite">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Nationalité :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nationalite">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Arduino">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>655</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Arduino Label</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>690</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>NOM</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CIN</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="text">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>690</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Rechercher :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>690</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
